packages/grafana-data/src/types/streamingDatasource.ts

```diff
@@ -0,0 +1,53 @@
+import { DataQueryResponseData } from './datasource';
```

Comment:
does this file need to be in `grafana-data`?  can we make it internal to the live package?